lib/upsert/column_definition.rb

```diff
@@ -36,7 +37,11 @@ def to_setter_arg
     end
 
     def to_setter
-      "#{quoted_name} = #{to_setter_value}"
+      if @increment
+        "#{quoted_name} = #{quoted_name} + 1"
```

Comment:
Not yet. I was doing this for a bit of an emergency MVP that was having some performance struggles while processing millions of records...this weekend is the first one I've gotten where I actually have some time to get back to it. I'll probably rewrite this tonight. I didn't like having to pass so much stuff around, so I thought I'd take another pass at it...I also thought about possibly doing an ```increment_by: value``` signature.